config.json

```diff
@@ -55,7 +55,7 @@
     "productpage_related_products_count": 10,
     "productpage_similar_by_views_count": 10,
     "categorypage_products_per_page": 12,
-    "shop_by_price_visible": true,
```

Comment:
I believe the existing configuration values for this field are saved in the TR DB, so renaming the field effectively wipes out all existing merchant settings, is that right?

Reply:
Thank you for pointing that out. I found a better way to implement the change without changing the TR DB value.